modules/go-compatibility-and-support-matrix.adoc

```diff
@@ -27,11 +27,11 @@ In the table, features are marked with the following statuses:
 
 s|Version s|kam  s|Argo CD CLI  s|Helm  s|Kustomize s|Argo CD s|Argo Rollouts s|Dex     s|RH SSO |
 
+|1.17.0 |NA  |2.14.7 TP |3.17.1 GA |5.6.0 GA |3.0.11 GA |1.8.3 GA |2.41.1 GA |7.6.0 GA |4.12-4.19
```

Comment:
Argo CD CLI must be `3.0.11` as well.

Reply:
Changed the Argo CD CLI version from 2.14.7 to 3.0.11 as per the feedback received..